### ISSUE TITLE
After upgrade Error: invalid output. Check your ruby interpreter settings

### ISSUE THREAD

**Issue by THPubs** [CraigWilliams/BeautifyRuby#81]
I added the ruby path, checked htmlbeautifier. Everything is fine but it's not working. It beautifies .rb files but not .html.erb files. Earlier it worked. But all of a sudden it stopped working. What might be the issue?

**Comment by pranavpr** [comment]
After reformatting my machine, I was facing the same issue. I was able to resolve the issue by installing an older version of htmlbeautifier gem:

```
gem install htmlbeautifier -v 0.0.12
```

**Comment by OwenBannister** [comment]
Awesome that fixed it, I also had to make sure I deleted all of my other versions of the htmlbeautifier gem

**Comment by THPubs** [comment]
Yes it worked but older version of htmlbeautifier stucks on big files
